test/integration/prerender/test/index.test.js

```diff
@@ -471,15 +475,26 @@ describe('SPR Prerender', () => {
   describe('export mode', () => {
     beforeAll(async () => {
       exportDir = join(appDir, 'out')
+      await fs.writeFile(
+        nextConfig,
+        `module.exports = { exportTrailingSlash: true }`
+      )
+      // TODO: investigate stale cache causing iSSG to break on export when a
+      // previous build was done with exportTrailingSlash disabled and then the
+      // following with it enabled
+      await fs.remove(join(appDir, '.next'))
```

Comment:
```suggestion
```
These lines can be removed after #9260 has been merged. This will fail if removed before that is merged